trio/tests/test_exports.py

```diff
@@ -18,3 +23,21 @@ def test_core_is_properly_reexported():
                 found += 1
         print(symbol, found)
         assert found == 1
+
+
+def test_pylint_sees_all_non_underscore_symbols_in_namespace():
+    # Test pylints ast to contain the same content as dir(trio)
+    linter = PyLinter()
+    ast_set = set(linter.get_ast(trio.__file__, 'trio'))
+    trio_set = set([symbol for symbol in dir(trio) if symbol[0] != '_'])
+    trio_set.remove('tests')
+    assert trio_set - ast_set == set([])
+
+
+def test_jedi_sees_all_completions():
+    # Test the jedi completion library get all in dir(trio)
+    script = jedi.Script(path=trio.__file__)
```

Comment:
I think the way `jedi.Script` is supposed to be used is maybe more like `jedi.Script("import trio\ntrio.")`? (As in, we imagine somebody typed those characters into a file and then hit tab to get a list of completions.)